gtest-parallel

```diff
@@ -465,7 +492,12 @@ def start_daemon(func):
 
 workers = [start_daemon(worker) for i in range(options.workers)]
 
-[t.join() for t in workers]
+try:
+  timeout.start()
```

Comment:
For paranoia reasons, it's probably better to start the timeout before you start the worker threads. So IOW, maybe move the "workers =" line into the try...except, below this line?